user_guide_src/source/incoming/incomingrequest/039.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+// foo=one&bar=two&baz[]=10&baz[]=20
+var_dump($request->getRawInputVar('bar'));
+
+// Outputs: two
+
+// foo=one&bar=two&baz[]=10&baz[]=20
+var_dump($request->getRawInputVar(['foo', 'bar']));
+
+/*
+ * Outputs:
+ * [
+ *      'foo' => 'one',
+ *      'bar' => 'two'
+ * ]
+ */
+
+// foo=one&bar=two&baz[]=10&baz[]=20
+var_dump($request->getRawInputVar('baz'));
+
+/*
+ * Outputs:
+ * [
+ *      10,
+ *      20
```

Comment:
Aren't these strings?

```suggestion
 *      '10',
 *      '20'
```